osu.Game/Tests/Visual/OsuTestScene.cs

```diff
@@ -118,7 +121,8 @@ public virtual void RecycleLocalStorage()
                 }
             }
 
-            localStorage = new Lazy<Storage>(() => new NativeStorage($"{GetType().Name}-{Guid.NewGuid()}"));
+            localStorage = new Lazy<Storage>(() => new NativeStorage(
+                Path.Combine(Path.GetDirectoryName(host.FullPath), $"{GetType().Name}-{Guid.NewGuid()}")));
```

Comment:
although it's very unlikely to happen, but this will fail if used before the `GameHost` is resolved, what do you think about using `AppDomain.CurrentDomain.BaseDirectory` instead? (which should work exactly the same way)

Reply:
please provide a case where that happens.